### DISCUSSION TITLE
[24 Gennaio 2023] - Cache

### DISCUSSION THREAD

**Discussion by alem1105** [CS-Swap/Architettura-degli-Elaboratori#13] [Cache]
### Traccia Esercizio

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/f47ae900-7dff-49f8-a5b4-4d42af4f2d5a)

<details><summary>Soluzione Prof</summary>
<p>

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/845da75d-3820-4128-bd60-2f33d3c3745c)

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/fddcb1d6-655d-47db-abe0-0da1330a068f)

</p>
</details>

**Comment by notedo** [comment]
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/33e8a96f-f4e0-4732-8e4e-0c0085b60cae)

**Comment by alem1105** [comment]
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/dc17111b-0646-4d3a-b0e2-e52daa970c03)
